samples/azure-quantum/simulators/SparseSimulator.ipynb

```diff
@@ -0,0 +1,2274 @@
+    "\n",
+    "Let's create a few groups (clusters) of qubits with GHZ state. Please execute the cell below. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "affiliated-value",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/x-qsharp-data": "[\"PrepareAndMeasureGhzState\"]",
+      "text/html": [
+       "<ul><li>PrepareAndMeasureGhzState</li></ul>"
+      ],
+      "text/plain": [
+       "PrepareAndMeasureGhzState"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "open Microsoft.Quantum.Canon;\n",
+    "open Microsoft.Quantum.Intrinsic;\n",
+    "open Microsoft.Quantum.Arrays;\n",
+    "open Microsoft.Quantum.Diagnostics;\n",
+    "open Microsoft.Quantum.Measurement;\n",
+    "\n",
+    "operation PrepareAndMeasureGhzState(nQubits : Int,   // Number of qubits to allocate (should be multiple of the nClusters parameter).\n",
+    "                                    nClusters : Int  // Number of groups (clusters) to break-up qubits into.\n",
+    "                                   ) : Unit {\n",
+    "    Message(\"Let's run a big simulation!\");\n",
+    "\n",
+    "    Message($\"Allocating {nQubits} qubits...\");\n",
+    "    use qs = Qubit[nQubits];\n",
+    "    Message(\"Finished allocating!\");\n",
+    "\n",
+    "    let clusterSize = nQubits / nClusters;          // Number of qubits in every cluster.\n",
+    "    Message($\"Creating {nClusters} clusters of GHZ states between {clusterSize} qubits...\");\n",
+    "    for i in 0..clusterSize..nQubits - 1 {          // Iterate through the clusters.\n",
+    "        let cluster = qs[i..(i + clusterSize - 1)]; // \n",
+    "        H(Head(cluster));                           // Apply H gate to the first-most qubit in the cluster.\n",
+    "        ApplyCNOTChain(cluster);                    // Apply CNOT gate to the remaining qubits in the cluster.\n",
+    "                                                    // After this, all qubits in the cluster will have the same state.\n",
+    "        Message($\"Finished cluster {i / clusterSize}.\");\n",
+    "    }\n",
+    "    Message(\"Finished GHZ clusters!\");\n",
+    "\n",
+    "    if nQubits < 15 {                               // For small states\n",
+    "        DumpMachine();                              // show the state.\n",
+    "    }\n",
+    "\n",
+    "    Message(\"Collapse and show final state:\");\n",
+    "    let results = MultiM(qs);\n",
+    "    Message($\"{results}\");\n",
+    "\n",
+    "    Message(\"Cleaning up...\");\n",
+    "    ResetAll(qs);\n",
+    "\n",
+    "    Message(\"Done!\");\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "flying-business",
+   "metadata": {},
+   "source": [
+    "Let's make the small states displayed conveniently on a hystogram. Please execute the cell below."
```

Comment:
```suggestion
    "Let's make the small states displayed conveniently on a histogram. Please execute the cell below."
```

Reply:
Similar here as well about "Please execute..." It would be good to follow the prose style for other notebook samples.